README.md

```diff
@@ -281,9 +281,13 @@ gsutil cp userdata.avro gs://${TEMP_GCS_BUCKET}
 ```
 
 ## Compile modules
+
 ### Prerequisites
-As we are using Testcontainers for unit testing, you need to install Docker on your machine. Please [see here the supported Docker versions](https://www.testcontainers.org/supported_docker_environment/).
-### Compile 
+
+As Testcontainers is used for unit testing, you need to install Docker on your machine. Please [see here the supported Docker versions](https://www.testcontainers.org/supported_docker_environment/).
```

Comment:
can you hyperlink to testcontainers.org link

Reply:
hyperlinked 'Testcontainers' to testcontainers.org also